articles/quickstart/webapp/java-spring-boot/01-login.md

```diff
@@ -52,11 +53,16 @@ If you are using Maven:
 <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.5.12</version>
+    <version>3.1.4</version>
```

Comment:
```suggestion
    <version>3.1.5</version>
```

This is the latest version and probably not necessary. We should try to upgrade when Spring Boot 3.2 comes out next month (Nov 23 - https://calendar.spring.io/)

Reply:
Just sharing for context, but I added the same review comments on the sample to ensure the versions are aligned. Ok to not update it, but if we update it here, we might want to update the sample as well.

https://github.com/auth0-samples/auth0-spring-boot-login-samples/pull/21#pullrequestreview-1694196401